### ISSUE TITLE
[CI] SearchableSnapshotsIntegTests classMethod failing (20 minute timeout)

### ISSUE THREAD

**Issue by joegallo** [elastic/elasticsearch#70894]
**Build scan:**
https://gradle-enterprise.elastic.co/s/papxgsqfogxna/tests/:x-pack:plugin:searchable-snapshots:internalClusterTest/org.elasticsearch.xpack.searchablesnapshots.SearchableSnapshotsIntegTests/classMethod

**Reproduction line:**
`null`

**Applicable branches:**
master

**Reproduces locally?:**
No

**Failure history:**
https://gradle-enterprise.elastic.co/scans/tests?tests.container=org.elasticsearch.xpack.searchablesnapshots.SearchableSnapshotsIntegTests&tests.test=classMethod

**Failure excerpt:**
```
  java.lang.Exception: Suite timeout exceeded (>= 1200000 msec).

  at __randomizedtesting.SeedInfo.seed([F0C610E0CB1C1AAB]:0)

```

**Comment by joegallo** [comment]
https://gradle-enterprise.elastic.co/scans/tests?search.relativeStartTime=P7D&search.timeZoneId=America/New_York&tests.container=org.elasticsearch.xpack.searchablesnapshots.SearchableSnapshotsIntegTests&tests.sortField=FAILED&tests.test=testCreateAndRestorePartialSearchableSnapshot&tests.unstableOnly=true

**Comment by ywelsch** [comment]
Fixed by #70876
